draft-demarco-status-attestations.md

```diff
@@ -40,38 +40,39 @@ informative:
 
 --- abstract
 
-Status Attestations are signed objects that demonstrate the validity status of a
-digital credential.
+Status Attestation is signed objects that demonstrate the validity status of a
+Digital Credential.
 These attestations are ephemeral and periodically provided
-to digital credential holders, that can present these to verifiers along
-with the corresponding digital credentials.
+to Holders, that can present these to Verifiers along
```

Comment:
@asharif1990 I used the defined terms after their definition, withing the dedicated section.

Abstract and introduction sections doesn't define nothing

I'm available to merge these but I'd like to ask you to get informed from other authors about this before merging

Reply:
Okay, you are right. In the abstract and intro, it is possible to avoid them. I wasn't familiar with IETF style. we can avoid the capitalized terms in the intro and abstract then